nextflow.config_old

```diff
@@ -0,0 +1,425 @@
+/*
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
TO be removed? What is this exactly?

Reply:
Oops, leftover of my debugging. Will remove.